test/test_distributions.py

```diff
@@ -8,7 +8,7 @@
 from common import TestCase, run_tests
 from torch.autograd import Variable, gradcheck
 from torch.distributions import (Bernoulli, Beta, Categorical, Dirichlet,
-                                 Exponential, Gamma, Laplace, Normal)
+                                 Exponential, Gamma, Laplace, Normal, MultivariateNormal)
```

Comment:
Could you also add some example parameters in `EXAMPLES` below, ideally one that specifies `cov` and another that specifies `scale_tril`?